crates/transparency/src/map/fork.rs

```diff
@@ -31,14 +31,14 @@ impl<D: SupportedDigest> Clone for Fork<D> {
 impl<D: SupportedDigest> Default for Fork<D> {
```

Comment:
I'm not sure we should have a default anymore. I can't think of a use for it and it'd be easy to use incorrectly.

Reply:
Can push this up with some other minor changes to address other comments.  Before I do, there are two places where `Fork::Default`  currently gets called.  Not sure that I see a way around making `Fork` fields public if we wanted to do this.  Not sure how we value the tradeoffs.

Reply:
In place of a `Default` which I think is ambiguous, you can add a specifically named constructor that creates the needed `Fork`. I'd do this for other defaults too. We should only have a `Default` if that has an actual meaning for a type.